autorest/azure/async.go

```diff
@@ -90,6 +90,27 @@ func (f *Future) Done(sender autorest.Sender) (bool, error) {
 	return false, err
 }
 
+// RetryAfter returns a duration the application should wait before checking
+// the status of the asynchronous request and true.  If a duration wasn't
+// specified then the function returns the zero-value time.Duration and false.
+func (f Future) RetryAfter() (time.Duration, bool) {
```

Comment:
>RetryAfter [](start = 16, length = 10)

Not the biggest fan of this name, (it sounds as if the retry happened inside), maybe GetWaitTime?

Reply:
Agreed, I'll fix up the name.

Reply:
Fixed.